### PR TITLE
[5G: MVC][SessionD] Accept-Release pipelined stub

### PR DESCRIPTION
**Pull request by ronit-kumar-acl**
Signed-off-by: ronit-kumar-acl <ronit.k@altencalsoftlabs.com>

## Summary

Contains - 
1. New proto definitions for response message from UPF to SessionD also includes periodic messages from UPF. (Provided by Wavelabs team)
2. Stub code in pipelined (rpc_receiver) to serve the Set Message coming from sessiond.
3. Changes in PipelinedClient.cpp to handle Response of set message to pipelined.
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by 119Vik** [comment]
RestartIntegrationTests

**Comment by 119Vik** [comment]
RestartIntegrationTests

**Comment by 119Vik** [comment]
RestartIntegrationTests

File: lte/protos/pipelined.proto [hunk 1]
```diff
@@ -377,8 +377,48 @@ message SessionSet {
...
+       PFCP_ENTRY_IN_CONGESTION = 12;
+       NO_RESOURCES_AVAILABLE = 13;
+       SERVICE_NOT_SUPPORTED = 14;
+       SYSTEM_FAILURE = 15;
+   }
+
+   CauseValues cause_ie=1;
+}
+
+message FailureRuleInformation {
```
Comment (themarwhal) [inline lte/protos/pipelined.proto:410]: Do we only have one failure per rule type at a time?

File: lte/protos/pipelined.proto [hunk 2]
```diff
@@ -377,8 +377,48 @@ message SessionSet {
 }
 
 //ToDo Response will reflect in upcoming proto
-message UpfRes {}
-
+message  UPFSessionContextState {
+    CauseIE cause_info = 1;
+    UPFSessionState session_snaphot = 2;
+    FailureRuleInformation failure_rule_id = 3; // The session and version which failed in UPF
```
Comment (themarwhal) [inline lte/protos/pipelined.proto:383]: `rule_install_failures` ? I think this name implies there is only one failure

File: lte/gateway/python/magma/pipelined/rpc_servicer.py [hunk 3]
```diff
@@ -533,7 +545,24 @@ def GetAllTableAssignments(self, request, context):
...
+                                                                sess_ver_no=request.sess_ver_no))
+             logging.info(response)
+             return response
+
+        else:
+            response =\
+                     UPFSessionContextState(cause_info=CauseIE(cause_ie=CauseIE.REQUEST_ACCEPTED),
+                                           session_snaphot=UPFSessionState(seid=request.seid,
+                                           sess_ver_no=request.sess_ver_no))
+            logging.info(response)
```
Comment (uri200) [inline lte/gateway/python/magma/pipelined/rpc_servicer.py:565]: is this not returning a response?

File: lte/gateway/python/magma/pipelined/rpc_servicer.py [hunk 4]
```diff
@@ -533,7 +545,24 @@ def GetAllTableAssignments(self, request, context):
...
+        
+        if request.state in [Fsm_state.CREATED, Fsm_state.ACTIVE, Fsm_state.RELEASE]:
+             response =\
+                        UPFSessionContextState(cause_info=CauseIE(cause_ie=CauseIE.REQUEST_ACCEPTED),
+                                              session_snaphot=UPFSessionState(seid=request.seid,
+                                                                sess_ver_no=request.sess_ver_no))
+             logging.info(response)
+             return response
+
+        else:
```
Comment (uri200) [inline lte/gateway/python/magma/pipelined/rpc_servicer.py:560]: maybe you can remove the else since you return inside the `if`

**Comment by ronit-kumar-acl** [comment]
Closing this as stub code is no longer required as we move on to integration test.

**Pull request closed without merging**